Flaky test

```bash
FAIL: test_filter_by_simplified_status (api.tests.test_declaration_controllers.TestDeclarationControllers.test_filter_by_simplified_status)
Il est possible de filtrer par le status simplifié
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/complements-alimentaires/complements-alimentaires/api/tests/utils.py", line 13, in authenticate_and_func
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/complements-alimentaires/complements-alimentaires/api/tests/test_declaration_controllers.py", line 174, in test_filter_by_simplified_status
    self.assertIn(ongoing_instruction_2.id, result_ids)
AssertionError: 188 not found in <generator object TestDeclarationControllers.test_filter_by_simplified_status.<locals>.<genexpr> at 0x7f2652310110>
```

--- a/api/tests/test_declaration_controllers.py
+++ b/api/tests/test_declaration_controllers.py
@@ -169,9 +169,7 @@ class TestDeclarationControllers(APITestCase):
 
         results = response.json()["results"]
         self.assertEqual(len(results), 2)
-        result_ids = (x["id"] for x in results)
-        self.assertIn(ongoing_instruction_1.id, result_ids)
-        self.assertIn(ongoing_instruction_2.id, result_ids)
+        self.assertCountEqual([x["id"] for x in results], [ongoing_instruction_1.id, ongoing_instruction_2.id])
 
         # Requête pour tous les deux
         response = self.client.get(

